api/src/main/java/org/pmiops/workbench/api/BigQueryService.java

```diff
@@ -39,7 +39,11 @@
 
   @Autowired private Provider<WorkbenchConfig> workbenchConfigProvider;
 
-  private BigQuery getBigQueryService() {
+  /**
+   * This method is protected, rather than private, so that it can be overridden for testing
```

Comment:
I'd replace this comment with `@VisibleForTesting`.

Reply:
I just replaced it. Thanks, Jay.